src/AddComment/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+
+import TextArea from '../TextArea';
+import Button from '../Button';
+
+import './style.css';
+
+class AddComment extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: '',
+      displaySuccess: false,
+      displayError: false
+    };
+  }
+
+  onChange = value => {
+    this.setState({ value });
+  };
+
+  onSubmit = event => {
+    event.preventDefault();
+    const { value } = this.state;
+    const { onCommentAdd } = this.props;
+
+    onCommentAdd({
+      variables: { body: value }
+    })
+      .then(({ data }) => {
+        this.setState({ displaySuccess: true });
+      })
+      .catch(error => {
+        this.setState({ displayError: true, errorMessage: error.toString() });
+      });
+  };
+
+  renderMessage = () => {
+    const { displaySuccess, displayError, errorMessage } = this.state;
+    if (displaySuccess) {
```

Comment:
Just in case you didn't know about it: Maybe the [enum conditional rendering](https://www.robinwieruch.de/conditional-rendering-react/#enums) could be used instead of all these if else statements. I guess in this case it would mean that the local state of the component needs to change slightly to be mapped to SUCCESS, ERROR, NONE.